examples/do-not-use/main.tf

```diff
@@ -789,19 +789,41 @@ locals {
 
 data "aws_iam_policy_document" "aws_load_balancer_controller" {
   statement {
-    resources = ["*"]
+    sid       = "AllowCreateServiceLinkedRole"
+    effect    = "Allow"
+    resources = ["arn:${local.partition}:iam::${local.account_id}:role/aws-service-role/elasticloadbalancing.amazonaws.com/AWSServiceRoleForElasticLoadBalancing"]
     actions   = ["iam:CreateServiceLinkedRole"]
 
     condition {
       test     = "StringEquals"
       variable = "iam:AWSServiceName"
-      values   = ["elasticloadbalancing.${local.dns_suffix}"]
+      values   = ["elasticloadbalancing.amazonaws.com"]
```

Comment:
why are we reverting?

Reply:
I added the policy with all the tfsec corrections that were in this [PR](https://github.com/aws-ia/terraform-aws-eks-blueprints/pull/1398), so it overwritten the suffix. Just fixed.